addon/components/high-charts.js

```diff
@@ -1,90 +1,105 @@
-import { assign } from '@ember/polyfills';
-import Component from '@ember/component';
+import Component from '@glimmer/component';
+
+import { tracked } from '@glimmer/tracking';
+import { action } from '@ember/object';
+
 import { getOwner } from '@ember/application';
-import { set, getProperties, get, computed } from '@ember/object';
 import { run } from '@ember/runloop';
+
+import buildOptions from '../utils/build-options';
 import { setDefaultHighChartOptions } from '../utils/option-loader';
 import { getSeriesMap, getSeriesChanges } from '../utils/chart-data';
-import layout from 'ember-highcharts/templates/components/high-charts';
-import merge from 'deepmerge';
 
-/* Map ember-highcharts modes to Highcharts methods
```

Comment:
This seems like it would still be useful right?

Reply:
restored!